docs/physics/old_physics.rst

```diff
@@ -0,0 +1,15 @@
+*********************
+Old Physics Section of the TARDIS Docu (Outdated)
```

Comment:
Old Physics Section (Outdated)

And make sure number of `*` you use both above and bottom are equal to the number of characters (including space) in the title - as per RST conventions